__tests__/app.test.js

```diff
@@ -53,3 +53,41 @@ describe("/api endpoints", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200 AND return the correct article object with the correct properties and status 200", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const { article } = response.body;
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+      });
```

Comment:
I can see why you approached it this way however, we should be more thorough as we know more about what we want to be returned therefore we can have more specific assertions. 

For example, we know what article_id we want and also we know what the data types of each key value should be. Have a look into .objectContaining and/or .toMatchObject to achieve this.